snuba/datasets/metrics_aggregate_processor.py

```diff
@@ -0,0 +1,182 @@
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import Any, Mapping, Optional, Sequence, Tuple
+
+from snuba import environment, settings
+from snuba.consumers.types import KafkaMessageMetadata
+from snuba.processor import (
+    AggregateInsertBatch,
+    MessageProcessor,
+    ProcessedMessage,
+    _ensure_valid_date,
+)
+from snuba.query.expressions import (
+    Expression,
+    FunctionCall,
+    Literal,
+    OptionalScalarType,
+)
+from snuba.utils.metrics.wrapper import MetricsWrapper
+
+snuba_metrics = MetricsWrapper(environment.metrics, "metrics.processor")
```

Comment:
could we name this `snuba_metrics.processor` just to be explicit?

Reply:
I *believe* it already gets prefixed with snuba, but my confidence in that is low:

https://github.com/getsentry/snuba/blob/bab9bc59f334fa8a7a19355dc14791492a9b885d/snuba/environment.py#L43-L46

Reply:
it does already get prefixed with `snuba`